spec/proto/runtime/v1/runtime.proto

```
@@ -4,6 +4,7 @@ package spec.proto.runtime.v1;
 
 import "google/protobuf/empty.proto";
 import "google/protobuf/any.proto";
+import "appcallback.proto";
```

buf

import "appcallback.proto": file does not exist

(COMPILE)